mail_channel_notify_tag/README.rst

```diff
@@ -0,0 +1,11 @@
+Mail Channel Notify Tag
+=======================
+
+This module re-enables channel_ids from the front end when the channels
```

Comment:
`channel_ids` is a technical term, this should be understandable to a non-technical person.